docs/source/guides/understanding_types_and_tags.ipynb

```diff
@@ -20,7 +20,7 @@
    "source": [
     "## Definitions of Types and Tags\n",
     "\n",
-    "Woodwork has been designed to allow users to easily specify additional information about data contained in a DataTable while providing interfaces to access the underlying data based on that additional information. Because a single DataTable might store various types of data like numbers, text, or dates in different columns, the additional information is defined on a per-column basis.\n",
+    "Woodwork has been designed to allow users to easily specify additional information about data contained in a DataFrame while providing interfaces to access the underlying data based on that additional information. Because a single DataFrame might store various types of data like numbers, text, or dates in different columns, the additional information is defined on a per-column basis.\n",
```

Comment:
I'm having a hard time parsing this first sentence--a lot of uses of the word "data". 

Specifically, "data contained in a DataFrame" feels kind of redundant, and users probably won't view Woodwork as an interface for _accessing_ underlying data since the DataFrame isn't being surrounded by another object.

My suggestion might be something like 

> Woodwork has been designed to allow users to easily specify additional typing information about a DataFrame while providing the ability to interface with that data based on that additional information.

Reply:
Good suggestion. I cleaned this up a bit.